dev/idm-am-common-user/idm/conf/repo.ds.json

```diff
@@ -0,0 +1,489 @@
+                        "ldapAttribute": "fr-idm-effectiveRole",
+                        "isMultiValued": true
+                    },
+                    "effectiveAssignments": {
+                        "type": "json",
+                        "ldapAttribute": "fr-idm-effectiveAssignment",
+                        "isMultiValued": true
+                    },
+                    "postalCode": {
+                        "type": "simple",
+                        "ldapAttribute": "postalCode"
+                    },
+                    "stateProvince": {
+                        "type": "simple",
+                        "ldapAttribute": "st"
+                    },
+                    "postalAddress": {
+                        "type": "simple",
+                        "ldapAttribute": "postalAddress"
+                    },
+                    "displayName": {
+                        "type": "simple",
+                        "ldapAttribute": "displayName"
+                    },
+                    "description": {
+                        "type": "simple",
+                        "ldapAttribute": "description"
+                    },
+                    "country": {
+                        "type": "simple",
+                        "ldapAttribute": "c"
+                    },
+                    "address2": {
+                        "type": "simple",
+                        "ldapAttribute": "postalAddress"
+                    },
+                    "city": {
+                        "type": "simple",
+                        "ldapAttribute": "l"
+                    },
+                    "givenName": {
+                        "type": "simple",
+                        "ldapAttribute": "givenName"
+                    },
+                    "sn": {
+                        "type": "simple",
+                        "ldapAttribute": "sn"
+                    },
+                    "telephoneNumber": {
+                        "type": "simple",
+                        "ldapAttribute": "telephoneNumber"
+                    },
+                    "mail": {
+                        "type": "simple",
+                        "ldapAttribute": "mail"
+                    },
+                    "organizationName": {
+                        "type": "simple",
+                        "ldapAttribute": "organizationName"
+                    },
+                    "lastSync": {
+                        "type": "json",
+                        "ldapAttribute": "fr-idm-lastSync"
+                    },
+                    "consentedMappings": {
+                        "type": "json",
+                        "ldapAttribute": "fr-idm-consentedMapping",
+                        "isMultiValued": true
+                    },
+                    "kbaInfo": {
+                        "type": "json",
+                        "ldapAttribute": "fr-idm-kbaInfo",
+                        "isMultiValued": true
```

Comment:
Worth pointing out that this "fr-idm-kbaInfo" entry is not the same as the one that AM would recognize (and is defined by the "kbaInfoContainer" objectClass, which is included here). Probably should align these.

Reply:
We have a jira for this.